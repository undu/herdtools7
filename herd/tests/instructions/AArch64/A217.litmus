AArch64 A217
(* MADD test *)

{
    0: X1 = 1 ;
    0: X2 = 2 ;
    0: X3 = 3 ;
}

P0 ;
MADD W0,W1,W2,W3;

forall (
    0: X0 = 5 /\
    0: X1 = 1 /\
    0: X2 = 2 /\
    0: X3 = 3
)
