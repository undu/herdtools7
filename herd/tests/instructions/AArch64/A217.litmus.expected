Test A217 Required
States 1
0:X0=5; 0:X1=1; 0:X2=2; 0:X3=3;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=5 /\ 0:X1=1 /\ 0:X2=2 /\ 0:X3=3)
Observation A217 Always 1 0
Hash=1e21f1ff7d0c97ef284d656651844b58

